ui/src/app/views/notebook-redirect/component.ts

```diff
@@ -121,7 +121,8 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
     this.creating = this.route.snapshot.data.creating;
 
     if (this.creating) {
-      this.notebookName = this.route.snapshot.queryParamMap.get('notebook-name');
+      this.notebookName =
+        decodeURIComponent(this.route.snapshot.queryParamMap.get('notebook-name'));
```

Comment:
Why would this change, was this a display bug?

Reply:
Um, think that's what it was I'd have to check again locally. Will do now.

Reply:
Yup, for sure. It displays the name with URI encoding in the Loading message